databases/db.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+
+const UserModel =require('./../models/users');
+const ChromeModel = require('./../models/chromes');
+
+
+const sequelize = new Sequelize('offside','root','',{
+    host:'127.0.0.1',
+    dialect:'mysql'
+});
```

Comment:
```suggestion
const sequelize = new Sequelize('offside','root','',{
    host:'127.0.0.1',
    dialect:'mysql'
});
```
Variables de entorno